src/main/java/com/smartbear/readyapi/plugin/git/ReadyApiGitIntegration.java

```diff
@@ -108,8 +108,9 @@ public void updateFromRemoteRepository(File projectFile, boolean b) {
 
             final Set<String> uncommittedChanges = gitObject.status().call().getUncommittedChanges();
             if (uncommittedChanges.size() > 0) {
-                UISupport.showErrorMessage("There are uncommitted changes, commit or revert back those changes before updating from remote repo");
-                return;
+                String message = "There are uncommitted changes, commit or revert back those changes before updating from remote repo";
```

Comment:
We should correct the grammar here a little bit: ``from remote repo`` => ``from the remote repo.``

And I'd even prefer ``repository`` instead of ``repo``.

Reply:
Done

Reply:
Great, thanks! For me the changes looks OK but you also requested a review from @Alexander-Paeshin. So let's wait for Alexander's approvement.